test/javascripts/components/global-filter/filtered-tags-chooser-test.js

```diff
@@ -0,0 +1,58 @@
+import { visit } from "@ember/test-helpers";
+import { acceptance, query } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+
+acceptance("Discourse Global Filter - Filtered Tags Chooser", function (needs) {
+  needs.settings({
+    discourse_global_filter_enabled: true,
+    global_filters: "support",
+    tagging_enabled: true,
+  });
+
+  needs.pretender((server, helper) => {
+    server.get("/tag/support/notifications", () =>
+      helper.response({
+        tag_notification: { id: "support", notification_level: 2 },
+      })
+    );
+
+    server.get("/tag/support/l/latest.json", () => {
+      return helper.response({
+        users: [],
+        primary_groups: [],
+        topic_list: {
+          can_create_topic: true,
+          draft: null,
+          draft_key: "new_topic",
+          draft_sequence: 1,
+          per_page: 30,
+          tags: [],
+          topics: [],
+        },
+      });
+    });
+
+    server.put("/global_filter/filter_tags/support/assign.json", () => {
```

Comment:
Similar question here, I wouldn't think you'd need to mock a request for `.../assign.json`.